docs/guide/agent-server-compatibility.asciidoc

```diff
@@ -0,0 +1,76 @@
+[[agent-server-compatibility]]
+== Agent/Server Compatibility
+
+Below is a chart that outlines the compatibility between different versions of the APM agents and the APM Server. 
+
+NOTE: APM Server version 6.5 introduced a new way for APM agents to communicate with the APM Server.
+To take advantage of new features,
+it is recommended that you update to the newest version of the applicable agent.
+
+[float]
+[[nodejs-compatibility]]
+==== Node.js Agent Compatibility
+
+[options="header"]
+|=======================================================================
+|Agent Version |APM Server Version
+|0.x |< 6.2
+|1.x |6.2-6.x
+|2.x |>= 6.5
+|=======================================================================
+
+[float]
+[[python-compatibility]]
+==== Python Agent Compatibility
+
+[options="header"]
+|=======================================================================
+|Agent Version |APM Server Version
+|1.x |6.1 beta[]
+|2.x/3.x |6.2-6.x
+|4.x |>= 6.5
+|=======================================================================
+
+[float]
+[[ruby-compatibility]]
+==== Ruby Agent Compatibility
+
+[options="header"]
+|=======================================================================
+|Agent Version |APM Server Version
+|1.x |\<= 6.x
```

Comment:
The ruby agent was in `beta` since January, and has gone GA in June.. I think we should align with the other agents to only show support starting with the APM Server version with which it was released (6.4 for ruby).